src/Blocks/BlockCactus.h

```diff
@@ -23,6 +23,39 @@ class cBlockCactusHandler :
 
 
 
+	/** Called before a cactus block is placed by a player, overrides cItemHandler::GetPlacementBlockTypeMeta().
+	Calls CanBeAt function to determine if a cactus block can be placed on a given block. */
+	bool GetPlacementBlockTypeMeta(
+		cChunkInterface & a_ChunkInterface,
+		cPlayer & a_Player,
+		const Vector3i a_PlacedBlockPos,
+		eBlockFace a_ClickedBlockFace,
+		const Vector3i a_CursorPos,
+		BLOCKTYPE & a_BlockType, NIBBLETYPE & a_BlockMeta
+	) override
+	{
+		if (a_Player.GetWorld()->DoWithChunkAt(a_PlacedBlockPos,
```

Comment:
Style: When using multi-line `if`, split line after `if(`

Reply:
Hopefully my last commits reflected the changes you wanted.